### ISSUE TITLE
props configured in route options not passed to react component on initial load

### ISSUE THREAD

**Issue by usrlocl** [framework7io/framework7#3788]
* Framework7 version: 6.0.0-beta.4
* React.js version: 17.0.0
* Platform and Target: Web
* Live Link or JSFiddle: NA

### Describe the bug
props configured in route options are not passed to react component on the initial route. They are however passed on subsequent navigation (by clicking link).

### To Reproduce
1. Configure props on initial route
```
    {
        path: '/',
        component: Home,
        options: {
           props: {
           foo: 'bar',
        }
     }

```
2. Load the main view with root url and push state enabled
```
   <View
        main
        url="/"
        browserHistory
        browserHistorySeparator=""
        browserHistoryRoot=""
      />

```
3. Access props in the Home component
```
    (props) => {
        console.log(props.foo);    // undefined on first load
    }

```
4. props.foo is undefined on first load
5. Navigate away and back to root.
6. props.foo is now available.

### Expected behavior
props should be available on each load of view.

**Comment by usrlocl** [comment]
Hi, I've raised a PR that fixes the issue on my local setup. Unfortunately I couldn't get a fiddle working against the exact scenario. It's a small fix though. Hopefully it should be obvious.

https://github.com/framework7io/framework7/pull/3789

**Comment by nolimits4web** [comment]
Fixed in #3789
